safe_cli/operators/hw_accounts/ledger_exceptions.py

```diff
@@ -3,19 +3,19 @@
 from ledgereth.exceptions import (
     LedgerAppNotOpened,
     LedgerCancel,
-    LedgerError,
     LedgerLocked,
     LedgerNotFound,
 )
 
 from safe_cli.operators.exceptions import HardwareWalletException
+from safe_cli.operators.hw_accounts.hw_account import InvalidDerivationPath
```

Comment:
Use relative imports, you are importing stuff from the same package

Why `InvalidDerivationPath` is not here? It should be either on `hw_accounts.exceptions` or `operators.exceptions` if you think it's a more generic exception